packages/contracts/package.json

```diff
@@ -29,6 +29,7 @@
     "test": "yarn run test:contracts",
     "test:contracts": "hardhat test --show-stack-traces",
     "test:gas": "hardhat test \"test/contracts/OVM/execution/OVM_StateManager.gas-spec.ts\" --no-compile --show-stack-traces",
+    "test:coverage": "NODE_OPTIONS=--max_old_space_size=8192 hardhat coverage",
```

Comment:
Solidity Coverage runs out of memory, at least on my machine, unless this `NODE_OPTIONS` prefix used.